### PR TITLE
[doc] Improve sdk page

### PR DESCRIPTION
**Pull request by nzw0301**
This PR fixes minor errors in the sdk page.

### CODE REVIEW

File: aim/sdk/objects/image.py [hunk 1]
```diff
@@ -25,9 +25,9 @@ class Image(CustomObject):
     Args:
          image (:obj:): pillow `Image` object or `torch.Tensor` or `numpy.array` used to construct `aim.Image`.
          caption (:obj:`str`, optional): Optional image caption. '' by default.
-         format (:obj: `str`, optional): Parameter for PIL's .save() method. 'png' by default.
-         quality (:obj: `int`, optional): Parameter for PIL's .save() method. 85 by default.
-         optimize (:obj: `bool`, optional): Parameter for PIL's .save() method. False by default.
+         format (:obj:`str`, optional): Parameter for PIL's .save() method. 'png' by default.
+         quality (:obj:`int`, optional): Parameter for PIL's .save() method. 90 by default.
```
Comment (nzw0301) [inline aim/sdk/objects/image.py:29]: Otherwise, I'll replace the default value `90` with `85`.

Comment (alberttorosyan) [reply aim/sdk/objects/image.py:29]: `90` is the correct value. Seems the docstring wasn't updated after the change.

**Review by alberttorosyan — APPROVED** [review]
Nice job! Thanks for contribution 🙌

File: aim/sdk/objects/image.py (hunk 1 above)
Comment (nzw0301) [reply aim/sdk/objects/image.py:29]: Thank you for your check!

**Pull request merged**